### ISSUE TITLE
The Text  on "Add Package" button and selected "nuget.org" dropdown elements are not having 4.5:1 color contrast ratio.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#277]
### Steps to Reproduce

1. Launch Xamarin Work application. It will open "Create a new workbook" window.
2. Navigate to File -> Add a Nuget package and activate it. Take a screenshot of opened popup having "nuget.org" element in selected state. (Refer Screenshot)
3. Check the colour contrast ration of "Add Package" button and "nuget.org" selected text in dropdown using the screenshot taken.

### Expected Behavior
Colour contrast ratio should be minimum 4.5:1

### Actual Behavior
Colour contrast ratio is lesser than 4.5:1
as shown in the attached screen-shots.

### User Impact:
Low vision user will feel difficulty in seeing the text.

### Environment
MAC: macOS High Sierra [10.13.4]
Workbook - 1.4 (master/b3f92f9) 

### Log File

![nuget 1](https://user-images.githubusercontent.com/22956130/39862511-94c2d430-5433-11e8-804d-b16c7faf1582.png)
![nuget 2](https://user-images.githubusercontent.com/22956130/39862514-99e43b3e-5433-11e8-9199-73a9ed5af4d7.png)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to this bug:
A11yMAS
Accessibility
MAS1.4.3
Xamarin-Workbook-MAC
Wipro

**Comment by abock** [comment]
This is a stock macOS control. We do not influence or control the rendering of this control. The system-wide "Graphite" appearance may exhibit 4.5:1 contrast. Regardless, this is not something we can or should address in our code as it comes from macOS itself.

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1833
